src/components/extensionEditor/attributeEditor/AttributeEditor.tsx

```diff
@@ -45,52 +39,51 @@ const AttributeEditor: React.FC<AttributeEditorProps> = ({
   let extensionElement = getElementById("Extension", "differential");
```

Comment:
doesn't `getElementById` always return the same type (like `ElementDefinition` or something) ? why isn't the function return type specified ?
If extensionElement is statically typed, you wouldn't have to specify "as string" or "as number" each time you mention it.

Reply:
Right, it always return an ElementDefinition fhir type.
I think we can type the output of the func with IElementDefinition